srcpkgs/serviio/files/serviio.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Is the desktop and png files shipped by them.
I really really not like this one.